### ISSUE TITLE
Cordova 8.0.0 compile fails

### ISSUE THREAD

**Issue by baversjo** [newrelic/newrelic-cordova-plugin#9]
Ran this command:
`cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git --variable IOS_APP_TOKEN="removed" --variable ANDROID_APP_TOKEN="removed"`

then `cordova run android`

I get a gradle errror:

```
$ cordova run android
Android Studio project detected
ANDROID_HOME=/Users/johan/Library/Android/sdk/
JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_77.jdk/Contents/Home
studio
Subproject Path: CordovaLib
Subproject Path: app

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/johan/projects/smrt-system/cordova/platforms/android/build.gradle' line: 63

* What went wrong:
A problem occurred evaluating root project 'android'.
> Could not find method compile() for arguments [com.newrelic.agent.android:android-agent:5.17.0] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
(node:56675) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: /Users/johan/projects/smrt-system/cordova/platforms/android/gradlew: Command failed with exit code 1 Error output:
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/johan/projects/smrt-system/cordova/platforms/android/build.gradle' line: 63

* What went wrong:
A problem occurred evaluating root project 'android'.
> Could not find method compile() for arguments [com.newrelic.agent.android:android-agent:5.17.0] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
(node:56675) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Is this project compatible with cordova 8? It is the current version of cordova on npm.

**Comment by clemmyn23** [comment]
#10 fixes this.  
It's to do with changed src directory structures in android 7

**Comment by wlin-incaps** [comment]
I fixed this by forking the repository and editing the android file with the gradle build path. We reported this issue to their dev team a few months ago, and the recommendation was to fix it on our own. Apparently it is fixed in their private repo, but not this public one.

File: hooks/android/android.js
From:
gradleBuildPath: path.join("platforms", "android", "app", "build.gradle"),
To:
gradleBuildPath: path.join("platforms", "android", "build.gradle"),
